Issue 648 make alias consistent

roopak-qlogic opened the pull request.

roopak-qlogic commented:
somehow appveyor build has been failed, and not getting exact reason.

coryan commented:
@roopak-qlogic we fixed the Windows/AppVeyor build problems, you need to rebase and push your changes again.

roopak-qlogic commented:
- First I tried to find line of each files containing "google::bigtable::admin::v2" and "namespace" ( using grep and pipe).
- got result with three distinct variable used "namespace btadmin", "namespace adminproto", "namespace admin_proto"
- So I changed adminproto and admin_proto with btadmin in all files, but making sure for following cases:
     - If btadmin variable is declared but not used in that file, then i removed completely from that file.
     - If btadmin is declared, but still "google::bigtable::admin::v2" is used directly in code, then I replaced "google::bigtable::admin::v2" 
       with btadmin variable.
  
above same steps used for btproto
